templates/footer.html

```diff
@@ -1,116 +1,13 @@
-        </li>
-        <li class="p-list__item">
-          <a class="p-link--soft" href="/blog/beyond-helm-and-kustomize-the-future-of-kubernetes-apps"><small>Helm alternative</small></a>
-        </li>
-        <li class="p-list__item">
-          <a class="p-link--soft" href="/blog/beyond-helm-and-kustomize-the-future-of-kubernetes-apps"><small>Kustomize alternative</small></a>
-        </li>
-        <li class="p-list__item">
-          <a class="p-link--soft" href="/blog/ansible-vs-terraform-vs-juju-fight-or-cooperation"><small>Ansible alternative</small></a>
-        </li>
-        <li class="p-list__item">
-          <a class="p-link--soft" href="/blog/juju-vs-infrastructure-as-code-tools"><small>Terraform alternative</small></a>
-        </li>
-      </ul>
-    </div>
-
-    <div class="col-3">
-      <h2 class="p-heading--5">Blog</h2>
-      <ul class="p-list">
-        <li class="p-list__item">
-          <a class="p-link--soft" href="/blog/ubuntu-and-canonical-at-kubecon-europe-2021"><small>Best Practices for creating charms</small></a>
-        </li>
-        <li class="p-list__item">
-          <a class="p-link--soft" href="/model-driven-operations-manifesto"><small>Model Driven Operations Manifesto</small></a>
-        </li>
-        <li class="p-list__item">
-          <a class="p-link--soft" href="/blog/beyond-helm-and-kustomize-the-future-of-kubernetes-apps"><small>Why Helm and Kustomize aren’t enough: the future of Kubernetes apps</small></a>
-        </li>
-        <li class="p-list__item">
-          <a class="p-heading--5 p-link--soft" href="/blog"><small>Read all of our blogs&nbsp;&rsaquo;</small></a>
-        </li>
-      </ul>
-    </div>
-  </div>
-
-  <div class="row">
-    <div class="u-sv2 u-fixed-width">
-      <hr>
-    </div>
-  </div>
-
-  <div class="p-strip is-shallow">
+<footer class="p-strip--dark footer">
+  <div class="p-section">
     <div class="row u-vertically-center">
-      <div class="col-6">
+      <div class="col-7">
         <h2>Join the community</h2>
       </div>
-
-      <div class="col-6">
-        <ul class="p-inline-list u-no-margin--bottom u-float-right--medium u-float-right--large">
-          <li class="p-inline-list__item">
-            <a href="https://chat.charmhub.io/charmhub/channels/juju" class="p-icon--chat"></a>
-          </li>
-          <li class="p-inline-list__item">
-            <a href="http://discourse.charmhub.io/" class="p-icon--forum"></a>
-          </li>
-          <li class="p-inline-list__item">
-            <a href="https://juju.is/careers" class="p-icon--careers"></a>
-          </li>
+      <div class="col-5">
+        <ul class="p-inline-list u-no-margin--bottom">
           <li class="p-inline-list__item">
-            <a href="https://github.com/canonical/operator/" class="p-icon--github"></a>
+            <a href="https://github.com/canonical/operator/" class="p-icon--github p-icon--github-white"></a>
```

Comment:
at this occasion, although not being part of the PR, the link to the https://www.youtube.com/jujucharms should be removed because it "auto-plays" a seven year old video. The video is nice, but mentions charms which have been deprecated.

Reply:
@mcjaeger is it better to link it to the operator day playlist?

Reply:
Yes that would be much ptter: https://www.youtube.com/playlist?list=PLwFSk464RMxkRa0kbUUllrDY6eZGWwn7j